packages/grafana-runtime/src/components/DataSourcePicker.tsx

```diff
@@ -22,8 +22,8 @@ import { ExpressionDatasourceRef } from './../utils/DataSourceWithBackend';
  * @internal
  */
 export interface DataSourcePickerProps {
-  onChange: (ds: DataSourceInstanceSettings) => void;
-  current: DataSourceRef | string | null; // uid
+  onChange?: (ds: DataSourceInstanceSettings) => void;
```

Comment:
I don't think we should change this. Without onChange the component doesn't really do much and developers aren't given immediate feedback via ts / intellisense to pass it.